HealthGPT/HealthGPT/HealthGPTView.swift

```diff
@@ -6,60 +6,88 @@
 // SPDX-License-Identifier: MIT
 //
 
-import HealthKit
-import OpenAI
 import SpeziFHIR
+import SpeziOpenAI
 import SpeziSecureStorage
 import SwiftUI
 
 struct HealthGPTView: View {
-    @AppStorage(StorageKeys.openAIModel) var openAIModel = Model.gpt3_5Turbo
-    @EnvironmentObject var secureStorage: SecureStorage<FHIR>
-    @State private var messages: [Message] = []
-
-    @State private var showAlert = false
-    @State private var alertText = ""
-
-    @StateObject private var messageManager = MessageManager()
+    @EnvironmentObject private var openAPIComponent: OpenAIComponent<FHIR>
+    @EnvironmentObject private var healthDataInterpreter: HealthDataInterpreter<FHIR>
+    
+    // I want to generate the main prompt here instead of initializing it as empty (from the health data interpreter)
+    @State private var messages: [Chat] = [] //try await healthDataInterpreter.generateMainPrompt()
```

Comment:
@rhea-mal @dguo8412 The best way to achieve this is to move the chat property in the `healthDataInterpreter` type. You can then create a function that loads the initial prompt and sub sequentially populates the chat there once the view appears or even better: the view appears and the onboarding has been complete.